geoips/plugins/yaml/sectors/dynamic/tc_web_2km_template.yaml

```diff
@@ -6,11 +6,23 @@ docstring: |
 
   Using pixel_width: 2000, pixel_height: 2000, num_lines: 700, num_samples: 700.
 spec:
-  sector_spec_generator:
-    name: center_coordinates
-    arguments:
-      projection: eqc
-      pixel_width: 2000
-      pixel_height: 2000
-      num_lines: 700
-      num_samples: 700
+  # sector_spec_generator:
+  #   name: center_coordinates
+  #   arguments:
+  #     projection: eqc
+  #     pixel_width: 2000
+  #     pixel_height: 2000
+  #     num_lines: 700
+  #     num_samples: 700
+  generators:
```

Comment:
Rolled this back because it was causing pytest to fail (though I think we are going to want something more like the part I left commented out in the end)